tests/test_calculator.hh

```diff
@@ -267,7 +267,19 @@ namespace rascal {
          {"smooth_width", {{"value", 0.5}, {"unit", "AA"}}}},
         {{"type", "Cosine"},
          {"cutoff", {{"value", 2.0}, {"unit", "AA"}}},
-         {"smooth_width", {{"value", 1.0}, {"unit", "AA"}}}}};
+         {"smooth_width", {{"value", 1.0}, {"unit", "AA"}}}},
+        {{"type", "RadialScaling"},
+         {"cutoff", {{"value", 4.0}, {"unit", "AA"}}},
+         {"smooth_width", {{"value", 0.5}, {"unit", "AA"}}},
+         {"rate", {{"value", .0}, {"unit", "AA"}}},
+         {"exponent", {{"value", 4}, {"unit", ""}}},
+         {"scale", {{"value", 2.5}, {"unit", "AA"}}}},
+        {{"type", "RadialScaling"},
```

Comment:
Just out of curiosity, what do the extra parameter sets do to the test runtime? I thought the gradient tests were relatively fast, but I don't remember the last time I ran them in debug.

Reply:
not much as I recall. the additional tests are restricted to the spherical expansion.

Reply:
the overall time for the tests is 7 sec in release